src/Psy/Command/ExternalEditorCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Psy\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExternalEditorCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('externaledit')
```

Comment:
I'd call the whole thing `edit` and skip the `externaledit` business entirely :)